cpp/vcpkg.json

```diff
@@ -31,8 +31,15 @@
       "features": [ "push", "pull" ]
     },
     "mongo-cxx-driver",
+    {
+      "name": "aws-c-io",
+      "version>=": "0.13.14",
```

Comment:
The newer baseline should give us these, no? We don't need an explicit constraint?

Reply:
Can remove. This was just so it's clear what the `"$version reason"` is commenting on